BreakpointSamples/Automation of Data Entry or Data Selection Processes/AutoUpdateLastCostExamples.vb

```diff
@@ -0,0 +1,30 @@
+Public Class AutoUpdateLastCostExamples
+
+  'Scenario: Automatically update last cost of products, whether a specific location, or generally
+  'Prerequisities: None
+  'Breakpoint: AfterStockTransferProcess and AfterManufactureIssueProcess
+
+  Public Sub Invoke(ByVal transaction As Sybiz.Vision.Platform.Inventory.Transaction.StockTransfer, ByVal e As Sybiz.Vision.Platform.Admin.Breakpoints.BreakpointEventArgs) 'Do not remove - SYBIZ
+
+    For each l As Sybiz.Vision.Platform.Inventory.Transaction.StockTransferLine In transaction.Lines
+      'If source location is elsewhere...
+      If l.SourceLocationDetails.Id = 0 Then
+      Sybiz.Vision.Platform.Inventory.Product.UpdateLocationLastCost(l.Product,l.Destination,l.UnitCost,true)
+      End If
+	  Next
+		
+	End Sub
+
+
+  Public Sub Invoke(ByVal transaction As Sybiz.Vision.Platform.Inventory.Transaction.ManufactureIssue, ByVal e As Sybiz.Vision.Platform.Admin.Breakpoints.BreakpointEventArgs) 'Do not remove - SYBIZ
+
+    For each l As Sybiz.Vision.Platform.Inventory.Transaction.ManufactureIssueLine In transaction.Lines
+      Sybiz.Vision.Platform.Inventory.Product.UpdateLastCost(l.Product,l.UnitCost,true)
```

Comment:
Not sure if there's anything else that needs considering in relation to manufacturing? Couldn't think of anything though...

Reply:
This is probably a good spot where you might want to not throw an exception, if doing a manufacture where a location is in negative stock this would explode and that might not be desired behavior; update last cost where possible and ignore it where it cannot.